### ISSUE TITLE
Bug in added profile fields

### ISSUE THREAD

**Issue by Lanchon** [ElJaviLuki/GrindrPlus#19]
version: v1.1 (for v8.17.1)

in the recent builds, added profile fields were re-enabled.
this triggers a nasty bug involving the BMI field when i view my own profile.

expected result: my profile says normal weight
actual result: damn thing says i'm overweight!
workaround: avoid looking at my own profile
lol im kidding :-p

**Comment by ElJaviLuki** [comment]
Are you sure you haven't gained weight? Do not blame others for not following the diet!

**Comment by ElJaviLuki** [comment]
> I wondered why all the people were all showing overweight.

Too sedentary.

**Comment by ElJaviLuki** [comment]
> version: v1.1 (for v8.17.1)
> 
> in the recent builds, added profile fields were re-enabled. this triggers a nasty bug involving the BMI field when i view my own profile.
> 
> expected result: my profile says normal weight actual result: damn thing says i'm overweight! workaround: avoid looking at my own profile lol im kidding :-p

My profile works fine in terms of BMI, hope you're kidding throughout your whole message and not just the part you explain your workaround lmao

**Comment by Lanchon** [comment]
im kidding, it works fine :)

i dont even put my stats in my profile.

but it would show overweight if i did: im well trained and BMI doesnt work for bodybuilders.

btw, it would be nice if the grid could be sorted by height or weight or BMI. i go for bulky guys and 90+% of guys have way low bmi for my taste.

**Comment by ElJaviLuki** [comment]
Yeah I'd be great but you know , there's a lot of work to be done in figuring out how the filter drawer UI works as its symbols are also obfuscated.
I don't know much about Android UI Drawers to be honest, if you get something new about it it could be awesome.

Also, the filters that are now server-side can also be implemented on the client-side by implementing the filtering on the hook

**Comment by ElJaviLuki** [comment]
> im kidding, it works fine :)
> 
> i dont even put my stats in my profile.
> 
> but it would show overweight if i did: im well trained and BMI doesnt work for bodybuilders.
> 
> btw, it would be nice if the grid could be sorted by height or weight or BMI. i go for bulky guys and 90+% of guys have way low bmi for my taste.

Well, after all, the BMI is just one more numeric reference, but not strictly enough to reach conclusions. If the guy in question catches your eye is more than enough haha. We're more than just numbers; although bankers, accountants or we coders take care of defending the opposite argument.
